bamboo/build/write_inject_vars.sh

```diff
@@ -6,19 +6,20 @@ set -e
 # REPO=${bamboo.REPO}
 
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+echo "$SCRIPT_DIR"
```

Comment:
Do we need this line?

```suggestion
```

Reply:
this was my solution to the unused variable warning- I can disable it instead if that's better

Reply:
Ah, I see. Yeah I think I'd prefer disabling it, or deleting the `SCRIPT_DIR` line.

Some of the scripts use `SCRIPT_DIR` to be sure of where we are in relation to Bamboo's file system, but if this script isn't using it then I feel confident we can delete it.